app/src/main-process/menu/build-default-menu.ts

```diff
@@ -219,6 +220,12 @@ export function buildDefaultMenu(): Electron.Menu {
         accelerator: 'CmdOrCtrl+Shift+F',
         click: emit('open-working-directory'),
       },
+      {
+        label: `Open in ${editorLabel}`,
+        id: 'open-external-editor',
+        accelerator: 'CmdOrCtrl+Shift+A',
```

Comment:
I'm open to suggestions for a better keyboard shortcut that doesn't clash.

Reply:
`CmdOrCtrl+Shift+A`?  Is that really the conventional keystroke shortcut for that command?  I seem to remember that command using an `E` in other apps where I've seen it (though I don't recall the modifier keys).